### ISSUE TITLE
Change all option 

### ISSUE THREAD

**Issue by bluestorm571** [SubtitleEdit/subtitleedit#5521]
Hi, 

I am currently using the OCR Repleace list to translate subtitles from Serbian to Croatian. In the tv show, there are many phrases which are used in every episode and those are not in the OCR Replace list. I thought "change all" option will add pair to OCR Replace list, but it doesn't.  It seems the only way to do it is to enter it manually in the "Options" menu. I am curious if it is possible to make an option to add pair to the user OCR list after pressing "change all" when spellchecking, so it is available for the next file as well. I think it is much easier that way than to manually enter it in the options menu.

**Comment by niksedk** [comment]
Items changed via `change all` are remembered for me...

**Comment by bluestorm571** [comment]
> Items changed via `change all` are remembered for me..

> Items changed via `change all` are remembered for me...

I tried to find multiple fixes like Fren -> Fran and I cannot find it in OCR Replace list. I am sure I pressed change all. 
Other words like Šefild --> Sheffield also are not on the list and therfore are not corrected in the next subtitle.

**Comment by niksedk** [comment]
I guess I need more info - screenshots always help. What OCR method are you using? What language/dictionary?

**Comment by bluestorm571** [comment]
Subtitles are in Serbian and I am translating into Croatian (similar to Serbian). Currently, I am following a tutorial from a website that suggests using subtitle edit OCR replace list as an easy way to replace many common words, i.e. Šefild to Sheffield.
So I open srt file and I go to Tools - Fix Common Errors and I choose the language to Croatian as well as Fix Common OCR Errors. 
After the process is done, I do the spellcheck with the Croatian language. As many words are not covered in the OCR list (there are no pairs for many words), I thought the option "change all" will add it, so next time I will have fewer words to correct. I am translating TV show, so words are repeating in almost every episode.
So, even though I added them to OCR list by pressing "Change all", in the next file those words are not replaced. So I go to Options - Settings - Wordlist and the words I just changed by pressing change all, are not on the list. Therefore, next time, I need to correct them manually again. 
![1](https://user-images.githubusercontent.com/94919823/143453860-f0ea0be7-fbaf-4299-be64-318d9baac6e8.png)
![2](https://user-images.githubusercontent.com/94919823/143453864-07c282eb-ac55-4df9-806e-34c0a48f70c9.png)
